### PR TITLE
feat: Add new member in UI #194

### LINKED ISSUE #194: Add new member in UI
https://forum.casbin.com/admin/member

image

Like node:

image

### PR DESCRIPTION
**Pull request by oranges-eating**
Fix: https://github.com/casbin/casnode/issues/194

Signed-off-by: killer <1533063601@qq.com>

### CODE REVIEW

**Comment by oranges-eating** [comment]
https://user-images.githubusercontent.com/53165082/119664094-92467f00-be65-11eb-881b-87d47d4aef40.mp4

**Comment by hsluoyz** [comment]
@kocoler @Kininaru plz review.

File: web/src/main/NewMember.js [hunk 1]
```diff
@@ -0,0 +1,279 @@
+import React from "react";
```
Comment (keainye) [inline web/src/main/NewMember.js:15]: Casbin license.

File: web/src/main/NewMember.js [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+class NewMember extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            form: {},
+            message: "",
+        };
+    }
+
+    componentDidMount() {
```
Comment (keainye) [inline web/src/main/NewMember.js]: This is an empty function.

File: controllers/member.go [hunk 3]
```diff
@@ -16,7 +16,6 @@ package controllers
 
 import (
 	"encoding/json"
-
```
Comment (keainye) [inline controllers/member.go]: No need to remove this. You can set your Go style like this: 

image

**Review by keainye — CHANGES_REQUESTED** [review]

File: controllers/member.go [hunk 4]
```diff
@@ -225,12 +225,25 @@ func (c *APIController) GetMemberLanguage() {
...
+	}
 	err := json.Unmarshal(c.Ctx.Input.RequestBody, &member)
 	if err != nil {
 		panic(err)
 	}
+	member.No = object.GetMemberNum() + 1
+	if object.AddMember(&member) {
+		resp = Response{Status: "ok", Msg: "true"}
+	} else {
+		resp = Response{Status: "fail", Msg: "Unauthorized."}
```
Comment (kocoler) [inline controllers/member.go]: Msg wrong.

File: controllers/member.go [hunk 5]
```diff
@@ -225,12 +225,25 @@ func (c *APIController) GetMemberLanguage() {
...
+		c.Data["json"] = resp
+		c.ServeJSON()
+		return
+	}
 	err := json.Unmarshal(c.Ctx.Input.RequestBody, &member)
 	if err != nil {
 		panic(err)
 	}
+	member.No = object.GetMemberNum() + 1
+	if object.AddMember(&member) {
```
Comment (kocoler) [inline controllers/member.go]: Missing `UploadAvatarToOSS`.

**Review by kocoler — APPROVED** [review]

**Review by keainye — APPROVED** [review]

**Comment by hsluoyz** [comment]
@oranges-eating plz provide a demo site and set my name as an admin.

**Comment by oranges-eating** [comment]
@hsluoyz https://forum.oranges.chromium.link/ 
usename:hsluoyz password:123

**Pull request merged**